### PR TITLE
Fix CheckPermission

### PR DESCRIPTION
**Pull request by Misaka-ZeroTwo**

### CODE REVIEW

**Comment by zhaguanyang** [comment]
https://discord.com/channels/656673194693885975/664263236048388097/1042882262708015204

**Review by IRacle1 — APPROVED** [review]

File: Exiled.Permissions/Extensions/Permissions.cs [hunk 1]
```diff
@@ -166,11 +166,20 @@ public static void Reload()
...
                 return true;
-            else if (sender is PlayerCommandSender && Player.Get(sender.SenderId) is Player player)
-                return player == Server.Host || player.CheckPermission(permission);
+            }
+            else if (sender is PlayerCommandSender)
+            {
+                Player player = Player.Get(sender.SenderId);
 
+                if (player is null)
+                    return false;
```
Comment (iopietro) [inline Exiled.Permissions/Extensions/Permissions.cs:178]:
```suggestion
                if (Player.Get(sender.SenderId) is not Player player)
                    return false;
```

**Review by iopietro — CHANGES_REQUESTED** [review]

**Comment by Thundermaker300** [comment]
PR should target dev branch, not master

**Comment by Thundermaker300** [comment]
`sender is GameCore.ConsoleCommandSender` what is `ConsoleCommandSender` in this case?

**Review by iopietro — APPROVED** [review]

File: Exiled.Permissions/Extensions/Permissions.cs [hunk 2]
```diff
@@ -172,11 +172,18 @@ public static void Reload()
         /// <returns>Returns a value indicating whether the user has the permission or not.</returns>
         public static bool CheckPermission(this CommandSender sender, string permission)
         {
-            if (sender.FullPermissions || sender is ServerConsoleSender)
+            if (sender.FullPermissions || sender is ServerConsoleSender || sender is GameCore.ConsoleCommandSender || sender == Server.Host.Sender)
```
Comment (iopietro) [inline Exiled.Permissions/Extensions/Permissions.cs:175]:
```suggestion
            if (sender.FullPermissions || sender is ServerConsoleSender || sender == Server.Host.Sender)
```
`GameCore.ConsoleCommandSender` doesn't exist anymore.

File: Exiled.Permissions/Extensions/Permissions.cs [hunk 3]
```diff
@@ -172,11 +172,18 @@ public static void Reload()
...
+            {
                 return true;
-            else if (sender is PlayerCommandSender && Player.Get(sender.SenderId) is Player player)
-                return player == Server.Host || player.CheckPermission(permission);
+            }
+            else if (sender is PlayerCommandSender)
```
Comment (iopietro) [inline Exiled.Permissions/Extensions/Permissions.cs:179]:
```suggestion
            else if (sender is PlayerCommandSender || sender is UserPrint)
```
Should this check be added to other ifs of `Exiled.Permissions` as well? `UserPrint` is the client-side player's console.

**Review by iopietro — CHANGES_REQUESTED** [review]

**Review by iopietro — APPROVED** [review]

**Pull request merged**